[2023-12-20 21:33:10,888] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: s3_to_postgres_dag.transform_read_s3 manual__2023-12-20T21:33:09.714979+00:00 [queued]>
[2023-12-20 21:33:10,898] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: s3_to_postgres_dag.transform_read_s3 manual__2023-12-20T21:33:09.714979+00:00 [queued]>
[2023-12-20 21:33:10,898] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2023-12-20 21:33:10,899] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2023-12-20 21:33:10,899] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2023-12-20 21:33:10,911] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): transform_read_s3> on 2023-12-20 21:33:09.714979+00:00
[2023-12-20 21:33:10,918] {standard_task_runner.py:52} INFO - Started process 11132 to run task
[2023-12-20 21:33:10,920] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 's3_to_postgres_dag', 'transform_read_s3', 'manual__2023-12-20T21:33:09.714979+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpzmqhokjl', '--error-file', '/tmp/tmpct1jdqdo']
[2023-12-20 21:33:10,921] {standard_task_runner.py:80} INFO - Job 29: Subtask transform_read_s3
[2023-12-20 21:33:10,976] {logging_mixin.py:109} INFO - Running <TaskInstance: s3_to_postgres_dag.transform_read_s3 manual__2023-12-20T21:33:09.714979+00:00 [running]> on host eddfb0464777
[2023-12-20 21:33:11,043] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=s3_to_postgres_dag
AIRFLOW_CTX_TASK_ID=transform_read_s3
AIRFLOW_CTX_EXECUTION_DATE=2023-12-20T21:33:09.714979+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-20T21:33:09.714979+00:00
[2023-12-20 21:33:11,055] {base_aws.py:190} INFO - Credentials retrieved from login
[2023-12-20 21:33:36,115] {python.py:175} INFO - Done. Returned value was: None
[2023-12-20 21:33:36,133] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=s3_to_postgres_dag, task_id=transform_read_s3, execution_date=20231220T213309, start_date=20231220T213310, end_date=20231220T213336
[2023-12-20 21:33:36,163] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-12-20 21:33:36,209] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
